package.json

```diff
@@ -35,6 +35,7 @@
     "@mapbox/eslint-config-mapbox": "^1.2.1",
     "@mapbox/mbx-assembly": "^0.28.2",
     "@mapbox/react-test-kitchen": "^0.1.3",
+    "@mapbox/web-analytics": "^0.5.1",
```

Comment:
Note: this is a private module. Including it makes it far easier to test segment events, but not easy for teammates who don't have access. I think we should leave for now for testing purposes, but can circle back if it becomes a blocker for others.

Reply:
Is this still an issue now that it's moved to optional dependencies?

Reply:
@colleenmcginnis now that it's an optional dependencies, I don't think non-aws users will have an issue install this repo locally.

Also want to note that web-analytics is installed here for local testing only (so we can make sure events are getting tracked) - it won't get bundled in the dr-ui build.